### ISSUE TITLE
UCI_Chess960

### ISSUE THREAD

**Issue by gbtami** [ecrucru/anticrux#12]
Seems anticrux incorrectly claims to support Chess960.

```
tamas@TAMAS:~/anticrux$ node anticrux-engine.js 
uci
id name AntiCrux 0.3.0
id author https://github.com/ecrucru/anticrux/
option name UCI_Chess960 type check default false
option name UCI_Variant type combo default suicide var suicide var antichess var giveaway
option name Skill Level type spin default 10 min 1 max 20
option name Debug type check default false
option name Precise Score type check default false
uciok
copyprotection ok
```

UCI spec say(*) engine should send it if it supports this variant. As I know anticrux doesn't play chess960.

(*)"If the engine knows about Chess960
it should send the command 'option name UCI_Chess960 type check default false'
to the GUI at program startup."

**Comment by gbtami** [comment]
It should list suicide960, antichess960 and giveaway960 in UCI_Variant option instead, IMO.

**Comment by ecrucru** [comment]
Hello,

Initially, this option aimed to tell any GUI that the engine is not specific to the classical start position. But you are right to mention that the claim is incorrect : the GUI will still continue to play the classical rules and this is in line with the specification.

So based on your information, I looked a bit further and the option has now disappeared (it was ignored anyway). I have updated the readme for WinBoard. For pyChess, it was requested in the ticket 1411 to improve the gameplay of that specific variant.

Thank you.
